tgstation.dme

```diff
@@ -3790,11 +3790,13 @@
 #include "modular_skyrat\code\modules\client\loadout\_medical.dm"
 #include "modular_skyrat\code\modules\client\loadout\_security.dm"
 #include "modular_skyrat\code\modules\client\loadout\backpack.dm"
+#include "modular_skyrat\code\modules\client\loadout\boxers.dm"
 #include "modular_skyrat\code\modules\client\loadout\hands.dm"
 #include "modular_skyrat\code\modules\client\loadout\head.dm"
 #include "modular_skyrat\code\modules\client\loadout\mask.dm"
 #include "modular_skyrat\code\modules\client\loadout\neck.dm"
-#include "modular_skyrat\code\modules\client\loadout\suit.dm"
```

Comment:
Bob, you accidentally unchecked modular suit loadout code.

Reply:
fuck lol